app/main.py

```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        {
+            "code": "name",
+            "text": "message",
+            "line_number": "line",
+            "column_number": "column",
+            "filename": "source"
+        }[signature]: ("flake8" if signature == "filename" else sense)
+        for signature, sense in error.items()
+        if signature != "physical_line"}
```

Comment:
you should extract specific pieces of information from the input dictionary by key